docs/reference-docs/rug/rug-conventions.md

```diff
@@ -28,22 +26,23 @@ concrete examples.
 
 [travis-editors]: https://github.com/atomist-rugs/travis-editors
 
-### Rug Archive
+### Rug Projects
```

Comment:
We should change it everywhere, but I don't think that means we shouldn't fix it as we go.